data_nutch_data_nci/conf/nutch-site.xml

```diff
@@ -45,7 +45,7 @@
 
 	<property>
 		<name>db.fetch.retry.max</name>
-		<value>30</value>
+		<value>10</value>
 		<description>NCI OVERRIDE: Things on the network can be sketchy, so I am upping this to a high value so that we do not accidently remove things that are having issues.  A value of 30 *should* basically mean that we should retry the item for about a month before skipping it.</description>
```

Comment:
Is this a deliberate change?  The comment says it should be a month.

Reply:
update to be 30